.github/workflows/test.yml

```diff
@@ -146,7 +146,17 @@ jobs:
     - name: Doctest Linux
       if: ${{ (runner.os == 'Linux') && always() }}
       shell: bash -l {0}
-      run: DGLBACKEND=pytorch pytest -v --ignore-glob='deepchem/**/test*.py' --doctest-modules deepchem --doctest-continue-on-failure
+      run: |
+        DGLBACKEND=pytorch pytest -v
```

Comment:
This isn't a proper fix; we are hardcoding skipping these files and it will be easy to leave these skips in place leaving these files without doctest tests for the long run. Let's discuss a more structured fix at OH